src/main/java/appeng/helpers/WirelessTerminalGuiObject.java

```diff
@@ -35,16 +35,15 @@
 import appeng.api.networking.IGridHost;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'appeng.api.networking.IGridHost'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AUselessImportCheck)